src/fs/mod.rs

```diff
@@ -116,6 +116,8 @@ pub use sendfile::sendfile;
 pub use special::*;
 #[cfg(linux_kernel)]
 pub use statx::statx;
+#[cfg(all(linux_kernel, any(target_env = "musl", target_env = "gnu")))]
+pub use statx::STATX_ATTR_MOUNT_ROOT;
```

Comment:
One thing we can do to make this code a little tidier is just change the `use statx::statx` above to be `use statx::*;`, instead of adding a new `use`, so that we don't have to repeat all the `cfg`s.